components/play-editor/element.js

```diff
@@ -59,7 +59,6 @@ export class MDNPlayEditor extends LitElement {
       });
       this._editor.setState(state);
     }
-    this.dispatchEvent(new Event("update", { bubbles: true, composed: true }));
```

Comment:
Yes, as I suspected, this causes the weird race condition to come back: I think it's to do with the order the components get loaded in. Let me debug a bit more, and either come up with a different solution for the reset, or a different solution to this race.

You should be able to reproduce by [loading up the kitchen sink](https://fred-pr549.review.mdn.allizom.net/en-US/docs/MDN/Kitchensink#try_it_3) and refreshing until you see:

<img width="794" height="398" alt="image" src="https://github.com/user-attachments/assets/04dad68d-1a12-4a13-9f64-174db588277f" />

Reply:
I was wrong, this is an unrelated issue: https://github.com/mdn/fred/issues/606

But I was right about this line being important, it ensures we save the state after formatting in the playground, however I've got a better solution for that in this PR: https://github.com/mdn/fred/pull/607